assign_phecodes.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+# coding: utf-8
+import pandas as pd
+import uuid
+import hail as hl
+from hail.utils import hadoop_open
+from ukbb_pan_ancestry import *
+from ukb_common import *
+hl.init()
+
+
+def checkpoint_tmp(hail_obj, tmppath='gs://ukbb-diverse-temp-30day/', tmpname=None, overwrite=True):
+    if tmpname is None:
+        tmpname = str(uuid.uuid4())
+    return hail_obj.checkpoint(tmppath + tmpname, overwrite=overwrite)
+
+
+# Read in the Phecode (v1.2b1) <-> ICD 9/10 codes mapping
```

Comment:
Can you wrap the bulk of the script in a main() function (and probably the hl.init() in there too)?

Relatedly, I'd probably suggest parameterizing overwrite (kinda like we do here: https://github.com/macarthur-lab/gnomad_qc/blob/master/annotations/generate_ld_data.py#L146 ), so that if someone runs this they don't accidentally overwrite data without explicitly asking it to do so.

Reply:
Fixed! Thanks for the review.